src/gui-commands.ts

```diff
@@ -124,9 +134,14 @@ export function registerGuiCommands(program: Command) {
   setCommand
     .command('password')
     .arguments('<password>')
-    .description('Set the GUI server password')
+    .description('Set the GUI server password, requirements: min 8 characters, at least 1 letter, at least 1 number, at least 1 special character')
```

Comment:
the list should be in this description as well so it shows up in the help. The user shouldn't have to put in an invalid password just to get the list